app/views/devise/registrations/new.html.erb

```diff
@@ -26,8 +26,8 @@
         </div>
 
 
-        <div class="form-actions">
-          <button class="log-in"><%= f.button :submit, "Sign up" %></button></b>
+        <div class="form-actions text-center">
+          <%= f.button :submit, class: "btn log-in" "Sign up" %>
```

Comment:
<%= f.button :submit, class: "btn log-in" "Sign up" %> => Why do we need "Sign up" here?